lib/board.rb

```diff
@@ -61,31 +103,29 @@ def get_row(position)
     position / board.size
   end
 
-  def win_with_columns?(mark)
-    board.transpose.any? { |column| same_symbol?(column, mark) }
-  end
-
-  def win_with_rows?(mark)
-    board.any? { |row| same_symbol?(row, mark) }
+  def backward_index(index)
+    board.size - index - 1
   end
 
-  def same_symbol?(rows, mark)
-    rows.all? { |row| row == mark }
+  def in_range(position)
+    if position < @POSITION_MIN || position > @POSITION_MAX
```

Comment:
I'm really curious about it. As the board size is set at its creation and you can't change it. Does it worth it to calculate each time we need its size (test/interface) throw a method?